pytorch_blade/src/torch-mlir/include/torch-mlir/Conversion/TorchToMhlo/MhloLegalizeUtils.h

```diff
@@ -113,10 +114,15 @@ void CreateReplaceOpAndInfer(
   rewriter.replaceOp(op, result->getResults());
 }
 
+std::vector<Value> getDimSizesOfTensor(
+    PatternRewriter& rewriter,
+    Operation* op,
+    const Value& value);
```

Comment:
nit: MLIR community does not encourage to  use `const`. 
https://mlir.llvm.org/getting_started/DeveloperGuide/

![image](https://user-images.githubusercontent.com/15364516/175023030-b10ee009-019f-40f0-8102-cfd03466b836.png)

Reply:
done